requirements.txt

```diff
@@ -28,7 +28,7 @@ scipy
 scikit-learn==1.2.2
 pynvml
 art
-fschat==0.2.36
+fschat @ git+https://github.com/lm-sys/FastChat.git@5095615810cf613dba7f27dd155f571fcff976d8
```

Comment:
Would it be possible to use a version instead of commit?

Reply:
<img width="245" alt="Screenshot 2024-04-21 at 4 55 02 PM" src="https://github.com/OpenAccess-AI-Collective/axolotl/assets/381258/d02d5c45-742c-4c32-b169-0aa2d5250e8e">
unfortunately, fastchat hasn't cut a release in a while too.